### PR TITLE
GH Pages and PDF CICD

### PR DESCRIPTION
**Pull request by revanthstrakz**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [x] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Manual test
- [x] 👍 yes
- [ ] 🙅 no

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [x] 🙅 no documentation needed

### CODE REVIEW

File: .github/workflows/pdf-docs-generation.yml
Comment (cpunt) [inline .github/workflows/pdf-docs-generation.yml:4]: Should we not just generate this on main branch to?

**Review by cpunt — COMMENTED** [review]
Looks great Revanth, just one comment!

File: .github/workflows/pdf-docs-generation.yml
Comment (revanthstrakz) [reply .github/workflows/pdf-docs-generation.yml:4]: yes good reminder i removed it as am working on cicd branch before

**Review by cpunt — COMMENTED** [review]
Lgtm! Before merging let's get @lePereT approval

**Review by lePereT — APPROVED** [review]
Thanks so much for this well-structured and thoughtful commit! Very much approve the changes you made to location of particular images, and the addition of a much needed favicon :)

**Pull request merged**
